### DISCUSSION TITLE
Environment variable for configuration of latitude and longitude

### DISCUSSION THREAD

**Discussion by Anmirazik** [jensrossbach/node-red-contrib-chronos#117] [Ideas & Requests]
Hi , is there a way for the config to get the latitude and longitude from environment variable ? The use cases is more on deploying the nodered instances in remote location , since remote location have different latitude and longitude , is there a way for the chronos config to receive input from environment variable instead of manually typing the latitude and longitude ? 

Im using flowfuse btw to deploy the nodered instances on remote locations and i would like to utilise the environment variable in flowfuse but then again chronos does not allow to receive input from environment variable , any help ? Thank you

**Comment by jensrossbach** [comment]
Hi @Anmirazik, currently the latitude and longitude can only be entered manually and there is no possibility to retrieve them from environment variables. I made some tests because Node-RED has some kind of built-in support for environment variables but unfortunately it doesn't work here (probably because the input fields for latitude and longitude are already verified on frontend side to be valid numbers).

Feel free to create a feature request in the issues section if you need this feature. But I cannot promise a quick implementation at the moment as I'm quite busy with other stuff.

**Reply by Anmirazik** [reply]
i see , thank you @jensrossbach . I will make a feature request for this in the issues section :) Thank you very much once again for this great library :)
